### ISSUE TITLE
Breaking template strings to multiple lines

### ISSUE THREAD

**Issue by wildeyes** [prettier/prettier#3397]
**Prettier 1.8.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBDTBeTAAzQBMAzAC1MxLUvIGtSya60GTMKzSHMfOlUgCt+VEqKET+vSeIbCBmACTAyECAF8aigOY1Mu0hX0CK-Wt1mVMac9VUAjbACdtZEro+7BXz+UNyHzJCAG4QABoQCAAHGABLdGRQbAxkMmwAGzQ4KMcXbDAGOBgAZRjC+KhdZBgXAFdckBIIMHSsnKiqnJcYAAUC3QBbbHbspuE0AA8AIQKiktLsIbgAGSq4Mc7o+pgY3YAmLaaKlx7kEGdHAE9M6EiQGJcqmAB1eJIYCmQADgAGKJPCA5V4FGK1BpNFxwACO9Xi0IG2GGoyQGXGURyQ3iEMamKqukycAAivUIPBjlEYNhHO9Pt8kAcqQV4pkCQBhCBDEYXKDQTZReo5AAqNLQx00miAA)

**Input:**
```jsx
const a = `sdfhsd dsfhdfksdf dsfskd hfsdk fsd fhsdj fhsdj fhsdj fsdkj fhsdkjfsd ${foo} djfg d gsdhg fsdh fdshfsdkjfh sh sd ${bar} fdgfdgd fgfdgdf gdfgdf`;
```

**Output:**
```jsx
const a = `sdfhsd dsfhdfksdf dsfskd hfsdk fsd fhsdj fhsdj fhsdj fsdkj fhsdkjfsd ${
  foo
} djfg d gsdhg fsdh fdshfsdkjfh sh sd ${bar} fdgfdgd fgfdgdf gdfgdf`;

```

**Expected behavior:**
I'd rather have an option to control such a thing (that is, disable this behaviour). My coworkers passionately dislike this, and that'll probably keep us from adopting Prettier despite it's superior excellency.

**Comment by MadaraUchiha** [comment]
Ideally, I'd like the opportunity to control where the break happens. In this instance, I'd rather introduce concatenation then break on template variable or on a square bracket accessor:
```js
const a = `bla bla bla bla bla ${Foo[
'bar'
]} bla bla bla`;
```

**Comment by azz** [comment]
This is fixed in 1.9! https://github.com/prettier/prettier/pull/3299

**Comment by ikatyang** [comment]
Closing as duplicate of #3315 or #3368.

---

**Prettier 1.9.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBDTBeTAAzQBMAzAC1MxLUvIGtSya60GTMKzSHMfOlUgCt+VEqKET+vSeIbCBmACTAyECAF8aigOY1Mu0hX0CK-Wt1mVMac9VUAjbACdtZEro+7BXz+UNyHzJCAG4QABoQCAAHGABLdGRQVxcIAHcABVcENGQQbAA3CHiSSIKMZDJsABs0OCjHF2wwBjgYAGUYlvioXWQYFwBXBpASCDAq2vqo3vqXGEzm3QBbbCm60eE0AA8AIWbW9o7sFbgAGV64DZnooZgY+4AmG9Hul3n850cATxrocoxFy9GAAdVKMAoyAAHAAGKJAiD1UHNGL5IFweaFa5RFxwACOQ3ieKW2FW6yQ1U2UXqK3iA2GozQvV0NTgAEUhhB4K8ojBsI5wSRIcgnnzmvEaiyAMIQFZrfJQaA4kBDeoAFQFeUp0zgmk0QA)

**Input:**
```jsx
const a = `sdfhsd dsfhdfksdf dsfskd hfsdk fsd fhsdj fhsdj fhsdj fsdkj fhsdkjfsd ${foo} djfg d gsdhg fsdh fdshfsdkjfh sh sd ${bar} fdgfdgd fgfdgdf gdfgdf`;
```

**Output:**
```jsx
const a = `sdfhsd dsfhdfksdf dsfskd hfsdk fsd fhsdj fhsdj fhsdj fsdkj fhsdkjfsd ${foo} djfg d gsdhg fsdh fdshfsdkjfh sh sd ${bar} fdgfdgd fgfdgdf gdfgdf`;

```
